### DISCUSSION TITLE
Is there a way to specify in the build command which external yaml file to use?

### DISCUSSION THREAD

**Discussion by syeda-git** [mkdocs/mkdocs#2769] [Q&A]
Hi,
There is one mkdocs.yml file. It references to multiple yaml files that includes variable definitions. If i need to produce Windows output, i include Windows variable file in mkdocs.yml file. If there is need to produce Linux output, i had to replace the instance of Window variable file with Linux variable file and then build.

When we set up automation (Jenkins ), the builds run unattended. So, wondering if there is a way to specify external yaml files as part of build parameters? There can be Windows build that would point to Windows variable file. 

Thanks...
Syed

**Comment by Andre601** [comment]
From what I know can you use `!ENV` in a mkdocs file to pull eviroment variables from the system.

Also, mkodcs files support inheritance from other files, but I doubt that this is what you are looking for.
